src/schemas/create-entity-operation-with-file-config.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "type": "object",
+  "properties": {
+    "operation": {
+      "type": "string",
+      "const": "create"
+    },
+    "uidField": {
+      "$ref": "uid-field-config.schema.json"
+    },
+    "save": {
+      "type": "boolean"
+    },
+    "returnEntity": {
+      "type": "boolean"
+    },
+    "dataJsonPath": {
+      "type": "string"
+    },
+    "responseModel": {
+      "type": "object",
+      "properties": {
+      }
+    },
+    "paramMatch": {
+      "type": "object",
+      "properties": {
+      }
+    },
+    "extensions": {
+      "type": "object",
+      "properties": {
+        "withDefaultValues": {
+          "type": "object",
+          "properties": {
+          }
+        }
+      }
+    },
+    "handleFile": {
+      "type": "object",
+      "properties": {
+        "path": {
+          "type": "string"
+        },
+        "name": {
```

Comment:
Have exportFields or destinationFields and maybe even call it sourceField for the other one. Also have directoryPath for saving files in...